mlflow/pipelines/pipeline.py

```diff
@@ -196,6 +196,18 @@ def get_artifact(self, artifact_name: str):
         """
         pass
 
+    @experimental
+    @abc.abstractmethod
+    def _get_artifact_path(self, artifact_name: str):
+        """
+        Read an artifact from pipeline output. artifact names can be obtained from
+        `Pipeline.inspect()` or `Pipeline.run()` output.
```

Comment:
This isn't quite accurate - this method returns the path to a pipeline artifact output. It does not read the artifact contents.